node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const input = `${process.argv[2]}`;
```

Comment:
don't forget the newline character here!

Reply:
correction made. 
![image](https://user-images.githubusercontent.com/110127607/204927603-3c67a401-caec-447b-a974-264583d0c858.png)